packages/ses/src/tame-regenerator-runtime.js

```diff
@@ -0,0 +1,16 @@
+import {
+  defineProperty,
+  iteratorPrototype,
+  iteratorSymbol,
+} from './commons.js';
+import { accessor, permitted } from './permits.js';
+
+export const tameRegeneratorRuntime = () => {
+  const iter = iteratorPrototype[iteratorSymbol];
+  permitted['%IteratorPrototype%']['@@iterator'] = accessor;
+  defineProperty(iteratorPrototype, iteratorSymbol, {
+    configurable: true,
+    get: () => iter,
+    set: () => true,
```

Comment:
This is more unsafe than it needs to be to solve the motivating problem. When the setter's `this` is exactly the expected prototype, this should fail silently. This does introduce an unsafety, but only to code that would otherwise have failed.

Aside from that case, it should do the remainder of the enable-property-override setter fix at
https://github.com/endojs/endo/blob/59b756eb896c72a75f4c0773f964903364bf7e71/packages/ses/src/enable-property-overrides.js#L108-L121

so that other assignment that should fail with a throw still do so, and that other assignments that should succeed still do.

Reply:
Since this is distinct from the general enablements mechanism. Feel free to omit the debug clause on lines 111..114 above